nextcord/ext/application_checks/errors.py

```diff
@@ -73,7 +73,17 @@ def __init__(
         self.checks: List[ApplicationCheckFailure] = checks
         self.errors: List[Callable[[Interaction], bool]] = errors
         super().__init__("You do not have permission to run this command.")
+        
+class ApplicationCheckAllFailure(ApplicationCheckFailure):
+    """Exception raised when at least 1 predicate in :func:`check_all` fails.
+
+    This inherits from :exc:`~.ApplicationCheckFailure`.
+    """
 
+    def __init__(
+        self,
+    ) -> None:
+        super().__init__("You do not have permission to run this command.")
```

Comment:
This should ideally be similar to check any in terms of information. This is not enough to work with ideally.